---
title: Synthetic Minimal Cells
subtitle: Primer to a PhD Thesis Project
image: /img/5_synells/synell-cropped.png
---
<style>
img {
  display: block;
  margin-left: auto;
  margin-right: auto;
  height: auto;
}
</style>


Synthetic minimal cells (synells) are microscopic bioreactors that allow scientists to study many of the molecules and processes of life. These artificial cells are built from the same materials as living cells (proteins and small molecules, encased in a phospholipid [liposome](https://en.wikipedia.org/wiki/Liposome)). But unlike regular cells, synells can be built to spec.

![Synells overview](/img/5_synells/synell1.png)

Scientists use synells to figure out how cells do what they do. Because they are built from scratch and don't have to contain _all_ the elements necessary for life, synells can isolate particular parts of biology that we are interested in studying. The synells we used in this paper, for example, contained the transcription/translation machinery (the hardware) from bacterial and mammalian cells, but none of the original DNA from those cells (the software).

Our project was an exploration of biological programming. We took internal hardware from living systems, added our own custom software (DNA that we designed and synthesized), and measured the outputs of our genetic programs. Previous research had done these experiments in test tubes or single liposomes. Our contribution is to show the versatility that comes from using liposomes to compartmentalize the various elements of a genetic circuit. 

Synells were part of my PhD thesis work, carried out in [Ed Boyden's group at the MIT Media Lab](http://syntheticneurobiology.org/). You can find the full paper in [Nature Chemistry](http://www.nature.com/nchem/journal/vaop/ncurrent/full/nchem.2644.html) (local copy [here](http://syntheticneurobiology.org/PDFs/16.11.adamala.pdf)).

## Common elements

All our synells throughout the paper contain transcription/translation (TX/TL) machinery from bacterial or mammalian cells. We added other molecules with useful behaviors, such as:

1. Activator molecules that triggered gene expression in our programs. We used two that can diffuse through the liposome membrane (arabinose, theophylline) and two that can't (IPTG, doxycycline).
2. Helper proteins (Tet, Lac) that mediate the activity of some of the activators.
3. Self-assembling membrane pores (aHL) that are wide enough to let IPTG and Dox pass into the liposomes. 
4. DNA encoding two different reporter molecules: firefly luciferase (fLuc) and Renilla luciferase (rLuc). We measured luciferase luminescence as the output from most of our genetic programs.
5. Membrane-anchored fusion peptides (SNARES) that can cause two liposomes to fuse with one another.

## Synells encapsulate genetic circuits

We started with the simple demonstration that a liposome can contain a genetic circuit and be activated by a remote trigger.

<img src="/img/5_synells/synell2.png" height="200" style="width:40%;"/>

Specifically, we built liposomes that expressed aHl and contained the DNA encoding for fLuc. We added Dox, which entered the liposome through aHL, interacted with Tet, and expressed the reporter fLuc.

We used this system to show how the crowded environment inside synells can promote multi-element reactions. We did this by expressing fLuc either as a single protein or as parts that have to come together after expression. The multi-component reactions were more efficient when carried out in liposomes, compared to bulk reactions in test tubes.

![](/img/5_synells/synell3.png)

## Synells enable parallel operations

Liposomes can isolate two genetic programs, such that they in neighboring liposome populations with different behavior, even in the same test tube.

<img src="/img/5_synells/synell4.png" height="250" style="width:50%;"/>

We created two synell populations that producer luciferase in response to Dox. We created them with very different amounts of aHL, so that one population would be much more permeable to Dox. 

![](/img/5_synells/synell5.png)

![](/img/5_synells/synell6.png)

Adding the same amount of Dox to the test tube created vastly different reactions from the two populations (as measured in relative light units, RLUs, of their respective luciferases).

<img src="/img/5_synells/synell7.png" height="300" style="width:60%;"/>

## Synells enable serial operations

Even more useful than communicating with the user is having synell populations communicate with each other. 

<img src="/img/5_synells/synell8.png" height="350" style="width:50%;"/>

For this demonstration, we built two populations of synells:

1. Sensors. These synells contain Dox (which can't get through the membrane) and contain DNA that responds to Theo (which can) to express aHL pores. Adding Theo to the test tube triggers aHL expression and Dox release.

2. Reporters. These synells express aHL from the start, and they contain DNA that responds to Dox to express fLuc. When the sensor synells release Dox, the reporter synells express fLuc.

![](/img/5_synells/synell9.png)

The interesting part about dividing a circuit between sensor and reporter synells is that those synells can contain vastly different machinery. In fact, we built sensors out of bacterial parts and reporters out of mammalian parts. These genetic elements are completely incompatible, but isolation into separate liposomes allows them to work together.

## Synells enable selective liposome fusion

Sometimes it's useful to keep circuit elements separate at first, before bringing them together at a specific time.

<img src="/img/5_synells/synell10.png" height="300" style="width:60%;"/>

SNAREs are a family of membrane-anchored proteins that trigger liposome fusion. They come in complementary pairs, such that a synell decorated with one kind of SNARE will fuse only with a synell decorated with the complementary SNARE.

![](/img/5_synells/synell11.png)

These proteins allow us to program liposomes to fuse in specific combinations. In our paper, we showed several examples of complementary circuit elements that could incubate separately before being brought together through SNARE fusion.

## In conclusion

Nature uses membranes to compartmentalize all manner of biochemical processes. That's the purpose of cell organelles and nuclei. Synells bring that same functionality to the design of artificial genetic circuits. The same biomolecules, when compartmentalized into separate synells, can perform different computations in the same test tube without interference. Synells thus represent a novel tool for synthetic biology, which will advance our ability to understand and recreate the natural world.

![](/img/5_synells/synell12.png)